File: bundle/nvim-treesitter/tests/indent/c/loop.c
void foo(int x)
{
    while (x > 0) {
        x--;
        continue;
    }

    for (int i = 0; i < 5; ++i) {
        x++;
        break;
    }

    do {
        x++;
    } while (x < 0);
}

File: bundle/nvim-treesitter/tests/indent/rust/loop.rs
fn foo(mut x: i32) {
    while x > 0 {
        x -= 1;
    }

    for i in 0..3 {
        x += 1;
    }

    loop {
        x += 1;

        if x < 100 {
            continue;
        }

        break;
    }
}
